package.json

```diff
@@ -15,5 +18,20 @@
     "node-ensure": "0.0.0",
     "underscore-plus": "^1.6",
     "atom-space-pen-views": "^2.0.3"
+  },
+  "configSchema": {
+    "reverseSyncBehaviour": {
+      "type": "string",
+      "enum": ["Disabled", "Click", "Double click"],
+      "default": "Click",
+      "title": "SyncTeX Reverse sync behaviour",
+      "description": "Specify the action on the PDF generated with the `--synctex=1` option that takes you to the source."
```

Comment:
The above changes seem unrelated to what this pull request is fixing. This pull request should be only about the deserializer, not about changing config options so that double-click is supported for synctex and not just click.